toolkit/scripts/get_config_deps.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
```

Comment:
I'd add `set -e` somewhere here, so we fail fast in case of an error. Or don't miss any errors by accident.